packages/pluggableWidgets/datagrid-web/package.json

```diff
@@ -32,6 +32,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.13.1",
     "@fortawesome/react-fontawesome": "^0.1.11",
     "@mendix/piw-utils-internal": "^1.0.0",
+    "deep-equal": "^2.0.2",
```

Comment:
You added a dependency but I don't see it being used?

Reply:
Missing dependency being delivered in settings.ts